src/quacc/recipes/newtonnet/ts.py

```diff
@@ -293,3 +307,250 @@ def _get_hessian(atoms: Atoms) -> NDArray:
+        write_xyz(output, symbols, smoother.path)
+    return symbols, smoother.path
+
+
+def setup_images(logdir: str, xyz_r_p: str, n_intermediate: int = 40):
+    """
+    Sets up intermediate images for NEB calculations between reactant and product states.
+
+    Parameters:
+    logdir (str): Directory to save the intermediate files.
+    xyz_r_p (str): Path to the XYZ file containing reactant and product structures.
+    n_intermediate (int): Number of intermediate images to generate.
+
+    Returns:
+    List: List of ASE Atoms objects with calculated energies and forces.
+    """
+    calc_defaults = {
+        "model_path": SETTINGS.NEWTONNET_MODEL_PATH,
+        "settings_path": SETTINGS.NEWTONNET_CONFIG_PATH,
+    }
+
+    calc_flags = recursive_dict_merge(calc_defaults, {})
+
+    try:
+        # Ensure the log directory exists
+        os.makedirs(logdir, exist_ok=True)
+
+        # Read reactant and product structures
+        reactant = read(xyz_r_p, index="0")
+        product = read(xyz_r_p, index="1")
+
+        # Optimize reactant and product structures using sella
+        for atom, name in zip([reactant, product], ["reactant", "product"]):
+            # atom.calc = calc()
+            atom.calc = NewtonNet(**calc_flags)
+            traj_file = os.path.join(logdir, f"{name}_opt.traj")
+            sella_wrapper(atom, traj_file=traj_file, sella_order=0)
+        # Save optimized reactant and product structures
+        r_p_path = os.path.join(logdir, "r_p.xyz")
+        write(r_p_path, [reactant.copy(), product.copy()])
+
+        # Generate intermediate images using geodesic interpolation
+        symbols, smoother_path = geodesic_interpolate_wrapper(
+            [reactant.copy(), product.copy()]
+        )
+        images = [Atoms(symbols=symbols, positions=conf) for conf in smoother_path]
+
+        # Calculate energies and forces for each intermediate image
+        for image in images:
+            # image.calc = calc()
+            # ml_calculator = calc()
+            image.calc = NewtonNet(**calc_flags)
+            ml_calculator = NewtonNet(**calc_flags)
+            ml_calculator.calculate(image)
+
+            energy = ml_calculator.results["energy"]
+            forces = ml_calculator.results["forces"]
+
+            image.info["energy"] = energy
+            image.arrays["forces"] = forces
+
+        # Save the geodesic path
+        geodesic_path = os.path.join(logdir, "geodesic_path.xyz")
+        write(geodesic_path, images)
+
+        return images
+
+    except Exception:
+        return []
+
+
+def run_neb_method(
+    method: str,
+    optimizer: Optimizer | None = NEBOptimizer,
+    opt_method: str | None = "aseneb",
+    precon: str | None = None,
+    logdir: str | None = None,
+    xyz_r_p: str | None = None,
+    n_intermediate: int | None = 20,
+    k: float | None = 0.1,
+    max_steps: int | None = 1000,
+    fmax_cutoff: float | None = 1e-2,
+) -> None:
```

Comment:
This should instead be added as `quacc.runners.ase.NEB` instead with a corresponding schema. That will also fix your problems with trying to write to custom directories.